slasher/detection/service.go

```diff
@@ -83,11 +83,11 @@ func (ds *Service) Start() {
 
 	// The detection service runs detection on all historical
 	// chain data since genesis.
-	go ds.detectHistoricalChainData(ds.ctx)
+	//go ds.detectHistoricalChainData(ds.ctx)
```

Comment:
why comment these out?

Reply:
Commented in Raul's PR. Not functioning at the moment and gets in the way of testing live detection.